### PR TITLE
Improve compare IPAddress in VM (IPScannerViewModel)#2569

### LINKED ISSUE #2569: Improve sort for IP address, OID, etc.
Improve sort in DataGrid Column for
- [x] IPv4/6
- [x] SNMP OID (Current order is: 1.1, 1.10, 1.2)

### PR DESCRIPTION
**Pull request by Ghislain1**
**Please provide some details about this pull request:**
- Usage of IPAddressComparer   by CollectionObservalbleCollection<>
- Remove Empty method such as LoadSettings()

Fixes #2569

**ToDo:**

- [ ] Update [documentation](https://github.com/BornToBeRoot/NETworkManager/tree/main/docs/Documentation) to reflect this changes
- [ ] Update [changelog](https://github.com/BornToBeRoot/NETworkManager/tree/main/docs/Changelog) to reflect this changes

---

**By submitting this pull request, I confirm the following:**

- [x] I have read and understood the [contributing guidelines](https://github.com/BornToBeRoot/NETworkManager/blob/main/CONTRIBUTING.md) and the [code of conduct](https://github.com/BornToBeRoot/NETworkManager/blob/main/CODE_OF_CONDUCT.md).
- [ ] I have have added my name, username or email to the [contributors](https://github.com/BornToBeRoot/NETworkManager/blob/main/Contributors.md) list or don't want to.
- [x] The code or resource is compatible with the [GNU General Public License v3.0](https://github.com/BornToBeRoot/NETworkManager/blob/main/LICENSE).

### CODE REVIEW

**Comment by BornToBeRoot** [comment]
Thanks! I will review it tomorrow

File: Source/NETworkManager/ViewModels/SNMPViewModel.cs [hunk 1]
```diff
@@ -22,6 +22,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 using NETworkManager.Localization;
+using ControlzEx.Standard;
+using Microsoft.VisualBasic.Devices;
```
Comment (BornToBeRoot) [inline Source/NETworkManager/ViewModels/SNMPViewModel.cs]: This namespaces shouldn't be required.

**Comment by BornToBeRoot** [comment]
Does this also work when you click on the column in the datagrid?

This is using the SortMemberPath which is currently an IP address as int32 (but i can't use it for IPv6)
image

**Comment by Ghislain1** [comment]
Yes It does on the column in the Datagrid
image

**Pull request merged**